tests/wsl/firstrun.pm

```diff
@@ -73,7 +73,13 @@ sub license {
             send_key 'alt-n';
         }
         # Accept license
-        assert_screen 'wsl-license';
+        # assert_screen 'wsl-license' or wsl-liscense-beta;
+        if (check_var("BETA", "1")) {
```

Comment:
`if (get_var("BETA", 0))`  might be better as BETA is not always defined (released products do not have it). So I am wondering whether you dont see warnings in os-autoinst.log in case of released products

Reply:
With BETA variable undefined, I ran: 

https://openqa.suse.de/tests/14342607
https://openqa.suse.de/tests/14342606 sled

Both are behaving as expected.